### ISSUE TITLE
Misaligned left/right panes in side-by-side display

### ISSUE THREAD

**Issue by jtkhong** [rtfpessoa/diff2html#301]
- OS: MacOS Mojave 10.14.6
- diff2html version: 2.12.2
- Using diff2html directly or using diff2html-ui helper: diff2html directly (getPrettyHtml)
- Extra flags: inputFormat: 'diff', drawFileList: false, matching: 'lines', outputFormat: 'side-by-side'
- Chrome: Version 79.0.3945.130 (Official Build) (64-bit)

### Step 1: Describe the problem:
Misaligned left/right panes in side-by-side display
#### Steps to reproduce:
1.  Load the snippet code into a Chrome browser

#### diff example:

```
<html>
<head>

<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/diff2html/2.12.2/diff2html.css">
<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/diff2html/2.12.2/diff2html.js"></script>
<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/diff2html/2.12.2/diff2html-ui.js"></script>

<script>
$(document).ready(function() {
    let str = "--- file1    2020-02-04 16:44:13.762578820 +0000\n" +
"+++ file1  2020-02-04 16:44:30.472911230 +0000\n" +
"@@ -13,10 +13,6 @@\n" +
"             disabled\n" +
"             order 2\n" +
"         }\n" +
"-        /Common/conciseui-ndcwest:vs_conciseui-ndcwest-80 {\n" +
"-            disabled\n" +
"-            order 3\n" +
"-        }\n" +
"     }\n" +
" }\n" +
" gtm pool /Common/p_conferencebridge_80 {\n" +
"@@ -31,5 +27,5 @@\n" +
"             order 1\n" +
"         }\n" +
"     }\n" +
"-    monitor /Common/http \n" +
"+    monitor /Common/https\n" +
" }";

    var diffHtml = Diff2Html.getPrettyHtml(str, {
        inputFormat: 'diff',
        drawFileList: false,
        matching: 'lines',
        outputFormat: 'side-by-side'
    });

    document.getElementById('compare').innerHTML = diffHtml;
});
</script>

</head>
<body>
    <div id="compare"></div>
    <hr/>
</body>
</html>
```

#### Observed Results:

- Please see screenshot
![Screen Shot 2020-02-04 at 12 05 32 PM](https://user-images.githubusercontent.com/14079307/73769571-e4333500-4748-11ea-8756-4f8bcadbc3f4.png)

#### Expected Results:

- The grey lines should lines up in both panes

#### Relevant Code:

```
// TODO(you): code here to reproduce the problem
```

**Comment by rtfpessoa** [comment]
This should be fixed in 3.0.0.
I will try to do a release soon

**Comment by jtkhong** [comment]
Thanks for the quick response!

**Comment by rtfpessoa** [comment]
Should be fixed in 3.0.0
